gammapy/obs/observation.py

```diff
@@ -43,6 +43,12 @@ class ObservationTable(Table):
     * ``OBS_ID``
     * ``ONTIME``
     * ``LIVETIME``
+    * ``TSTART``
+    * ``TSTOP``
+    * ``AZ``
+    * ``ALT``
+    * ``RA``
+    * ``DEC``
```

Comment:
I thought so. I wasn't sure however how to make the proper link. Thanks for the tip.